src/index.ts

```diff
@@ -188,17 +204,23 @@ async function announceWinner(
     )) as TextChannel;
 
     const emoji = contestType === 'meme' ? '🎉' : '🦴';
-    const contest = contestType === 'meme' ? 'Meme de la semana' : 'Hueso de la semana';
-
+    const contest =
+        contestType === 'meme'
+            ? 'Meme de la semana'
+            : contestType === 'bone'
+                ? 'Hueso de la semana'
+                : contestType === 'medal'
+                    ? 'Medalla de la semana'
+                    : 'Tipo de concurso no definido';
```

Comment:
You can use a `switch` statement instead of nested ternary operators to make this more readable:

Reply:
```
const contestMap = {
    meme: 'Meme de la semana',
    bone: 'Hueso de la semana',
    medal: 'Medalla de la semana',
    default: 'Tipo de concurso no definido'
};

const contest = contestMap[contestType] || contestMap.default;
```

Reply:
```
const contestMap = {
    meme: 'Meme de la semana',
    bone: 'Hueso de la semana',
    medal: 'Medalla de la semana',
    default: 'Tipo de concurso no definido'
};

const contest = contestMap[contestType] || contestMap.default;
```

Reply:
Even Better!